### PR TITLE
Enable container tests for SL Micro on PublicCloud

### PR DESCRIPTION
**Pull request by mimi1vx**
POO: https://progress.opensuse.org/issues/155944

VR: http://quasar.suse.cz/tests/46# and http://quasar.suse.cz/tests/47#

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]

File: tests/publiccloud/slem_prepare.pm [hunk 1]
```diff
@@ -23,7 +23,7 @@ sub run {
     if (get_var("PUBLIC_CLOUD_CONTAINERS")) {
         my $runtime = get_required_var('CONTAINER_RUNTIMES');
         # Install packages for container test runs
-        trup_call("pkg install $runtime toolbox");
+        trup_call("pkg install $runtime toolbox cni-plugins");
```
Comment (mloviska) [inline tests/publiccloud/slem_prepare.pm:26]: shouldn't we drop cni testing when it comes to sle-micro 6.0+?

Comment (mimi1vx) [reply tests/publiccloud/slem_prepare.pm:26]: IDK ? without this package results are red: http://quasar.suse.cz/tests/26

Comment (mloviska) [reply tests/publiccloud/slem_prepare.pm:26]: ok, this is 5.5. So what versions of sle-micro and sl-micro we should run this test against?

By default, cni is not used, but netavark -> http://quasar.suse.cz/tests/26#step/podman/20

Either we have a test bug, that something is mangling the network backend, or there is a product bug

Comment (mimi1vx) [reply tests/publiccloud/slem_prepare.pm:26]: Oh

**Review by pdostal — APPROVED** [review]

**Review by asmorodskyi — APPROVED** [review]
LGTM

**Pull request merged**